src/OpenFeature.Contrib.Providers.Flipt/Flipt.Rest.Client.cs

```diff
@@ -0,0 +1,6796 @@
+//----------------------
+// <auto-generated>
+//     Generated using the NSwag toolchain v14.1.0.0 (NJsonSchema v11.0.2.0 (Newtonsoft.Json v13.0.0.0)) (http://NSwag.org)
+// </auto-generated>
+//----------------------
```

Comment:
I know this is a bit tedious, but for security and maintenance reasons, I'd prefer to generate this as part of the build rather than commit it. Is that possible?